### DISCUSSION TITLE
Excecuting a sequecnce of scene switches without looping or overriding others

### DISCUSSION THREAD

**Discussion by eriksenchee** [WarmUpTill/SceneSwitcher#454] [Q&A]
yo! I just got into using this amazing plugin and I'm trying to figure out how to do a specific function.

I want to basically make a sequence of scene switching and I want to have it go down the list of sequences until the end and then STOP. When using the sequence tab, the sequences will often loop or if a sequence is assigned to be excecuted earlier it will often do that instead (unless i'm doing something wrong). I was wondering if I could get any advice or direction for this purpose. I'm hoping to make files that hold these strings of sequences that I can load up and run for a specfic thing. 

To summarize: I need some help figuring out how to use the plugin to excute a list of sequences in order and then stop when it reaches the end.

Thanks alot!

**Comment by WarmUpTill** [comment]
> I need some help figuring out how to use the plugin to excute a list of sequences in order and then stop when it reaches the end.

It would probably best to setup a macro / multiple macros similar to the example described here:
https://github.com/WarmUpTill/SceneSwitcher/wiki/Automatically-cycle-through-a-list-of-scenes

The example above uses a condition that is always true.
Meaning the sequence of scene switches will always be active, which is not something you want.
So you will have to replace the condition with something that better fits your needs.

You could for example use a hotkey to trigger the scene switches similar to this example:
https://github.com/WarmUpTill/SceneSwitcher/wiki/Advance-through-a-list-of-scenes-by-hotkey

So a mixture of the two shared example might get you to what you want to achieve.
If you needs further advice on this feel free to let me know! :)

>I'm hoping to make files that hold these strings of sequences that I can load up and run for a specfic thing.

That is currently not supported, unfortunately.
If you need to provide a text file as input for the instructions it would probably easiest to set up a custom script instead of relying on this plugin.

**Reply by eriksenchee** [reply]
yo! thank you for showing me the macro, this is exactly what I needed. you have made an amazing plugin and i really appreciate your work! thanks again
